config/locales/views/en.yml

```diff
@@ -0,0 +1,47 @@
+en:
+  application:
```

Comment:
* `application` is the place to store locales used in the `app/views/application/*`
* `shared` is nested to the `devise`
* for email locales we could use same `views` prefix since mailer templates are the stored in the `app/views`